### PR TITLE
[ClassificationStore] Provide group and key models

### PR DESCRIPTION
**Pull request by kjkooistra-youwe**
Allowing easier access to active group and key data.

### CODE REVIEW

**Comment by kjkooistra-youwe** [comment]
Using this for several projects. Figured it would be useful to have this in the core. Any feedback is welcome.

**Comment by weisswurstkanone** [comment]
related: https://github.com/pimcore/pimcore/issues/5129

File: models/DataObject/Classificationstore/Key.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    /**
+     * @param string|null $language
+     * @param bool $ignoreFallbackLanguage
+     * @param bool $ignoreDefaultLanguage
+     * @return mixed
+     */
+    public function getValue(
+        ?string $language = 'default',
+        $ignoreFallbackLanguage = false,
+        $ignoreDefaultLanguage = false
```
Comment (jdreesen) [inline models/DataObject/Classificationstore/Key.php:73]:
```suggestion
        bool $ignoreFallbackLanguage = false,
        bool $ignoreDefaultLanguage = false
```

File: doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/15_Classification_Store.md [hunk 2]
```diff
@@ -128,6 +128,45 @@ $allValues = $store->getItems();
...
+            $keyConfiguration->getName(),
+            $keyConfiguration->getTitle(),
+            $value,
+            ($key->getFieldDefinition() instanceof QuantityValue),
+        ]);
+    }
+}
+```
+
+The `Key::getValue()` method supports the language, ignoreFallbackLanguage and ignoreDefaultLanguage arguments.
```
Comment (jdreesen) [inline doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/15_Classification_Store.md:162]:
```suggestion
The `Key::getValue()` method supports the `language`, `ignoreFallbackLanguage` and `ignoreDefaultLanguage` arguments.
```

**Review by jdreesen — COMMENTED** [review]
Some suggestions

**Comment by weisswurstkanone** [comment]
thanks a lot!

**Pull request merged**